tests/plugins/widget/manual/insertparagraph.md

```diff
@@ -0,0 +1,32 @@
+@bender-tags: widget, faeture, 4.16.0, 4467
```

Comment:
Please bump version tag.

Reply:
Increased to 4.16.1. I don't know if you're using semantic versioning...